src/serialbox-python/pp_ser/pp_ser.py

```diff
@@ -136,33 +141,33 @@ def __reg_shortcuts(self, shortcut):
         l = []
         if re.match('(^$|[IJK][IJK1-9]*)', shortcut):
             if shortcut == '':
-                l = '1 1 1 1 0 0 0 0 0 0 0 0'.split()
+                l = '1 0 0 0 0 0 0 0 0 0 0 0'.split()
```

Comment:
So scalars are 1D arrays whose only dimension size is 1?

Reply:
yes